### DISCUSSION TITLE
一张图实现柱状图在x轴上方与下方并在上面将二者的差距用面积图实现

### DISCUSSION THREAD

**Discussion by shengminhe** [antvis/G2#5108] [Q&A]
阿里云效中缺陷收敛：
![微信图片_20230529122956](https://github.com/antvis/G2/assets/59502108/8a93f214-4e2b-4463-87ba-a7ebbbc1414e)
一张图实现柱状图在x轴上方与下方并在上面将二者的差距用面积图实现

**Comment by hustcc** [comment]
在这个 [DEMO](https://g2.antv.antgroup.com/examples/general/interval#bar-stacked-diverging-rounded) 基础上，在加一个 area 图叠加就可以了，我晚点弄个数据试试。

**Reply by shengminhe** [reply]
怎么通过typeScript实现这个demo

**Reply by shengminhe** [reply]
怎么去实现吖

**Reply by hustcc** [reply]
在这个 [demo](https://g2.antv.antgroup.com/zh/examples/general/dual/#pareto) 中把 chart.line 换成 chart.area 即可。

**Reply by shengminhe** [reply]
感谢您的回复，这个demo可能还是无法满足上述的图表，望百忙之中能在后续解答我的疑问。

**Reply by pearmini** [reply]
可以参考这里：https://github.com/antvis/G2/pull/5148/files#diff-3ef1130ce11e1d68e5483901757a0e89df432e10e256430ffa093adf56f74971
